WBooks/Screens/BookLibrary/BooksTableView.swift

```diff
@@ -11,11 +11,11 @@ import WolmoCore
 
 class BooksTableView: UITableViewCell, NibLoadable {
     
-    @IBOutlet weak var table: UITableView! {
-        didSet {
+    @IBOutlet weak var table: UITableView!
+       func configureLibraryTableView() {
             table.backgroundColor = .backgroundLightBlue()
             table.separatorStyle = .none
+            let nib = UINib.init(nibName: "BookCell", bundle: nil)
```

Comment:
couldn't you use `table.register(cell: BookCell.self)` ?

Reply:
You mean using table.register(cell: BookCell.self)
instead of
let nib = UINib.init(nibName: "BookCell", bundle: nil)
table.register(nib, forCellReuseIdentifier: "BookCell")

Because its not filling the library with only that line